api/v1alpha1/openapi.yaml

```diff
@@ -3741,23 +3741,71 @@ components:
             - $ref: '#/components/schemas/ImageApplicationProviderSpec'
             - $ref: '#/components/schemas/InlineApplicationProviderSpec'
     ImageApplicationProviderSpec:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/ApplicationVolumeProviderSpec'
+        - type: object
+          properties:
+            image:
+              type: string
+              description: Reference to the container image for the application package.
+          required:
+            - image
+    InlineApplicationProviderSpec:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/ApplicationVolumeProviderSpec'
+        - type: object
+          properties:
+            inline:
+              type: array
+              description: A list of application content.
+              items:
+                $ref: "#/components/schemas/ApplicationContent"
+          required:
+            - inline
+    ApplicationVolumeProviderSpec:
+      type: object
+      properties:
+        volumes:
+          type: array
+          description: List of application volumes.
+          items:
+            $ref: "#/components/schemas/ApplicationVolume"
+    ApplicationVolume:
+      allOf:
+        - type: object
+          description: Defines a named volume used by an application.
```

Comment:
`ApplicationVolume` defines the `name` property twice.
Once here, as an optional prop. Second as part of `ImageVolumeProviderSpec`

Reply:
yikes good catch